src/mappers/provinces/province_mapper.cpp

```diff
@@ -49,4 +49,16 @@ std::set<std::string> mappers::ProvinceMapper::GetEquivalentVic3Provinces(const
    }
 
    return equivalent_provinces;
+}
+
+
+void mappers::ProvinceMapper::CheckAllVic3ProvincesMapped(const std::vector<std::string>& vic3_provinces) const
+{
+   for (const auto& vic3_province: vic3_provinces)
+   {
+      if (!vic3_to_hoi4_province_map_.contains(vic3_province))
```

Comment:
just a mental note for later, you should probably ignore unmapped lakes or wastelands, but that would allow some logic to iterate over actual provinces.

Reply:
I tend to insist on explicit mappings to nothing in such cases.